src/appComponents/tracks/TracksController.js

```diff
@@ -15,29 +17,28 @@ tracksModule.config(["$stateProvider", function($stateProvider) {
 
 }]);
 
-tracksModule.controller("TracksController", ["$mdDialog","$sessionStorage", "$rootScope", "ApiJsonFactory",
+tracksModule.controller("TracksController", ["$mdDialog", "$sessionStorage", "$rootScope", "ApiJsonFactory",
         function ($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
-          var tsc = this;
+          let tsc = this;
 
-          if ($sessionStorage.tracks === null || 
-            typeof $sessionStorage.tracks === "undefined")
+         if ($sessionStorage.tracks === null || typeof $sessionStorage.tracks === "undefined")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Opening curly brace does not appear on the same line as controlling statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2431821597/issues/source?bid=3233857&fileBranchId=3238420#l24)